tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -197,26 +197,34 @@ Start Workbench
 
 Access To Workbench
     [Documentation]    Gets access to a workbench from DS Project details page after it got launched
-    [Arguments]    ${username}=${TEST_USER.USERNAME}  ${password}=${TEST_USER.PASSWORD}  ${auth_type}=${TEST_USER.AUTH_TYPE}
-    Switch Window   NEW
+    [Arguments]    ${username}  ${password}  ${auth_type}
+    # Switch Window   NEW
```

Comment:
Can you remove this line if it's not needed?

Reply:
done